docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -1,24 +1,23 @@
-[[docker-machine-driver-install]]
-= Installing Docker Machine drivers
+[[set-up-hypervisor-driver-plugin]]
+= Setting up the hypervisor driver plugin
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
 
 toc::[]
 
-[[docker-machine-drivers-overview]]
+[[hypervisor-drivers-overview]]
 == Overview
-
 Minishift uses Docker Machine and its driver plugin architecture to
-provide a consistent way to manage the OpenShift VM. Minishift embeds
-VirtualBox and VMware Fusion drivers so no additional steps are required to use them.
+provide a consistent way to manage the OpenShift VM.
+Minishift embeds VirtualBox and VMware Fusion drivers so no additional steps are required to use them.
 
-However, other drivers require manual installation steps that are
+However, other drivers require manual installation and setup that are
 described in the following sections.
 
 [[kvm-driver-install]]
-== KVM driver
+=== KVM driver
```

Comment:
Adding triple `=` sign, remove it from table of content list. Check, the `View` button of this page.